servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1187,6 +1187,23 @@ public static Completable completed() {
         return CompletedCompletable.INSTANCE;
     }
 
+    /**
+     * Creates a {@link Completable} which when subscribed will invoke {@link Runnable#run()} on the passed
+     * {@link Runnable} and emit the value returned by that invocation from the returned {@link Completable}. Any error
+     * emitted by the {@link Runnable} will terminate the returned {@link Completable} with the same error.
+     * <p />
```

Comment:
```suggestion
     * <p>
```

Reply:
Ok.  I thought it was strange you asked me to use `<p/>`, but I went with it.  :-)

Reply:
woops, ya sorry about that